packages/scss/src/components/_action-icon.scss

```diff
@@ -64,24 +51,71 @@
 	&.mod-invert {
 		color: _component("actionIcon.invert.color");
 
-		&:hover, &:focus {
-			&:not([disabled]) {
+		&:not([disabled]) {
+			&:hover, &:focus {
 				background-color: rgba(255, 255, 255, .1);
 				color: _component("actionIcon.invert.color");
 			}
+
+			&:active {
+				background-color: rgba(255, 255, 255, .2);
+			}
+		}
+	}
+
+	&.mod-outlined {
+		box-shadow: 0 0 0 1px _color("grey", 400);
+
+		& + & {
+			margin-left: 6px;
```

Comment:
where this number come from ?

Reply:
It's to cover the specific case of mod-outline where we have a 4px outline on focus/active state. 
Without this margin, the outline effect spread over nearby buttons.